src/components/eip3643/admin/TokenInfo.tsx

```diff
@@ -36,54 +33,31 @@ type TokenNameItem = {
 export default function TokenInfo({
   tokenSelected,
 }: {
-  tokenSelected: TokenNameItem | null;
+  tokenSelected: TokenNameItem;
 }) {
   const { accountEvm } = useWalletInterface();
-  const [balance, setBalance] = useState("");
-
+  const { data: tokenDecimals, isPending: tokenDecimalsPending } =
+    useReadTokenDecimals(tokenSelected.address);
   const { data, mutateAsync: mint, error, isPending } = useMintToken();
 
+  //@TODO separate out mint form from token info
   const form = useFormik({
     initialValues: {
       address: accountEvm,
-      value: "10",
+      value: `${DEFAULT_TOKEN_MINT_AMOUNT}`,
```

Comment:
`${DEFAULT_TOKEN_MINT_AMOUNT}` => `DEFAULT_TOKEN_MINT_AMOUNT?.toString()`
Would be better in this way, but it's almost the same.

Reply:
Any arguments why it would be better?